### DISCUSSION TITLE
Use on macOS

### DISCUSSION THREAD

**Discussion by lpil** [Difegue/Stylophone#83] [Q&A] [answered]
Hello! Thank you for this program 💜 

Is it possible to use it on OSX as well as padOS and iOS?

Thanks,
Louis

**Comment by Difegue** [comment, marked as answer]
👋 Glad you like it! If you have an apple silicon machine, you should be able to just use the iOS/iPadOS version as-is on macOS by grabbing it from the App Store.  

If you're still on Intel... Well I might have a mac catalyst version available eventually.

**Reply by lpil** [reply]
Thank you very much! I didn't realise it was possible to use iOS apps on macOS like that.
